step.yml

```diff
@@ -65,6 +65,15 @@ inputs:
 
         Call `xcodebuild -help` for available export options.
       category: "app/pkg export configs"
+  - xcodebuild_options:
```

Comment:
Please move the input after *workdir*. The order counts inside a category, and this is a less important one. You can check how it looks using `bitrise :workflow-editor`.

Reply:
moved it to the end, looks good in the workflow editor